### DISCUSSION TITLE
JDBC driver and profiling

### DISCUSSION THREAD

**Discussion by solita-kattelus** [duckdb/duckdb#14100] [General]
I noticed I can enable profiling also in jdbc by doing sql pragma calls. With fast look it is nice and output goes err stream or something? Is there any api to access profiling output in jdbc driver or listener approach or anything to get kind of control where profiling information goes. E.g. I would like to myself log it in certain way and level according to what is query taking time etc...

**Comment by Tishj** [comment]
You could set `profile_output`
description: `The file to which profile output should be saved, or empty to print to the terminal`
